packages/modules/src/modules/banners/choiceCardsButtonsBanner/ChoiceCardsButtonsBannerBlue.tsx

```diff
@@ -0,0 +1,51 @@
+import { css } from '@emotion/react';
+import { brand } from '@guardian/src-foundations';
+import React from 'react';
+import { validatedBannerWrapper, bannerWrapper } from '../common/BannerWrapper';
+import { BannerRenderProps } from '../common/types';
+import { ChoiceCardsBanner } from './ChoiceCardsButtonsBanner';
+
+const bannerId = 'choice-cards-buttons-banner-blue';
+export const backgroundColor = '#F1F8FC';
+export const headingColor = brand[400];
+export const borderTopColorStyle = css`
+    border-top: 1px solid ${brand[400]};
+`;
+
+const ChoiceCardsBannerBlue = ({
+    onCloseClick,
+    content,
+    choiceCardAmounts,
+    countryCode,
+    submitComponentEvent,
+    tracking,
+    numArticles,
+    isSupporter,
+    separateArticleCount,
+}: BannerRenderProps) => {
+    return (
+        <ChoiceCardsBanner
```

Comment:
Could both <ChoiceCardBanner> components be merged into a single one? Perhaps if your going to loose either the tab or button version after the abtest its not worth doing?

Reply:
Which components did you mean could be merged into one here? After the abtest we'd settle on one or the other